README.md

```diff
@@ -13,17 +13,45 @@ SCI is a program to identify sub-compartments from HiC data. SCI utilizes graph
 * python 2.7
 
 **Python Libraries**
-* [scikit-learn] >=0.19.0 
+* [scikit-learn] >=0.19.0, <= 0.20
 * [Numpy] >= 1.15
```

Comment:
forgot to update the dependency version on numpy, the last numpy release tosupport Python 2.7: see https://docs.scipy.org/doc/numpy/release.html#numpy-1-16-0-release-notes

should be: [Numpy] <= 1.16